open-budget-il/index.yaml

```diff
@@ -114,6 +114,12 @@ indexes:
 
```

Comment:
this file is still modified somehow... let's revert it and remove the 'autogenerated' tag as you suggested.